src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2899,21 +2899,34 @@ void SessionImpl::saveResumeData()
         ++m_numResumeData;
     }
 
+    // clear queued storage move jobs except the current ongoing one
+    while (m_moveStorageQueue.size() > 1)
+        m_moveStorageQueue.removeLast();
```

Comment:
Why not just resize it to 1 if it is greater than 1?

Reply:
I mean to use single `resize()` instead of multiple `removeLast()`.

Reply:
Qt5 does not have `QList::resize()`. I replaced it with something else.